src/gui/main.cpp

```diff
@@ -82,6 +85,12 @@ int main(int argc, char **argv)
     QQuickStyle::setStyle(style);
     QQuickStyle::setFallbackStyle(QStringLiteral("Fusion"));
 
+#if defined Q_OS_WIN
+    if (QOperatingSystemVersion::current().version() <= QOperatingSystemVersion::Windows11.version()) {
```

Comment:
This includes Windows 11, do I read this correctly?

Reply:
yes
thanks
not intended